Editor/Editors/CinemachineFreeLookEditor.cs

```diff
@@ -30,6 +30,8 @@ protected override void OnEnable()
         {
             base.OnEnable();
             Target.UpdateInputAxisProvider();
+            
+            CinemachineSceneToolUtility.RegisterTool(CinemachineSceneTool.TrackedObjectOffset);
```

Comment:
Why are you registering?  You are not doing anything with it.  Also, you are not handling lens tools.